src/index.html

```diff
@@ -11,7 +11,213 @@
+            Password:
+            <input
+              type="password"
+              name="password"
+              minlength="5"
+              maxlength="32"
+              required
+              autocomplete="off"
+            >
+          </label>
+        </div>
+      </fieldset>
+      <fieldset class="fieldset">
+        <legend>
+          An intereating fact about you!
+        </legend>
+        <div class="form-field">
+          <label>
+            Do you love cats?
+            <label>
+              <input
+                type="radio"
+                name="love cats?"
+                value="yes"
+                required
+              >
+              Yes
+            </label>
+            <label>
+              <input
+                type="radio"
+                name="love cats?"
+                value="no"
+                required
+              >
+              No
+            </label>
+          </label>
+        </div>
+        <div class="form-field">
+          <label>
+            What is your favorite color?
+            <input
+              type="color"
+              value="#000"
+              name="favorite color"
+              required
+            >
+          </label>
+        </div>
+        <div class="form-field">
+          <label>
+            What time do you go to bed?
+            <input
+              type="time"
+              name="sleep time"
+              required
+            >
+          </label>
+        </div>
+        <div class="form-field">
+          <label>
+            What are your favorite brand of cars?
+            <select
+              name="favorite car"
+              multiple
+              required
+            >
+              <option value="BMW">BMW</option>
+              <option value="Audi">Audi</option>
+              <option value="Lada">Lada</option>
+            </select>
+          </label>
+        </div>
+        <div class="form-field">
+          <label>
+            How do you rate our work?
+            <input
+              type="range"
+              min="0"
+              max="10"
+              value="0"
+              name="work rate"
+              required
+            >
+          </label>
+        </div>
+      </fieldset>
+      <fieldset class="fieldset">
+        <legend>
+          Additional info:
+        </legend>
+        <div class="form-field">
+          <label>
+            Comments:
+            <textarea name="Comments"></textarea>
+          </label>
+        </div>
+        <div class="form-field">
+          <label>
+            Would you recommend us?
+            <select
+              name="recommend?"
+              required
+            >
+              <option
+                value="yes"
+                selected
+              >
+                Yes
+              </option>
+              <option
+                value="no"
+              >
+                No
+              </option>
+            </select>
```

Comment:
Only if tag have more than 2 attributes use multiline 
```suggestion
            <select name="recommend?" required>
              <option value="yes" selected>Yes</option>
              <option value="no">No</option>
            </select>
```